### ISSUE TITLE
Allow to choose user-preferred romanization system. 

### ISSUE THREAD

**Issue by ThomasLeigh** [SYtor/Kanji-Dojo#54]
Recently romanization of some characters have been changed within the app, partially due to avoiding confusion between Kana characters pronounced/romanized the very same way. However, changing "ji" (じ) into "zi" may give much confusion, especially for English students, because many may find "ji" much more intuitive pronunciation-wise than "zi".

Therefore, given different romanization schools may fit various students, it may be very helpful to allow users to choose their own preferred romanization style within the app's settings (e.g., Hepburn, Nihon-shiki, Kunrei-shiki - see the implementation here: https://github.com/theblackwidower/KanaQuiz ) . Would You be willing to do so?

**Comment by syt0r** [comment]
Interesting, thanks for the info, I didn't know there were multiple kana transliteration systems. Unfortunately, some of them can make it hard to distinguish characters so I think it's better to have a clear single system, but maybe it can be improved by adding additional notes like "X, reads as Y"

**Comment by ThomasLeigh** [comment]
> I think it's better to have a clear single system

I agree. Unfortunately the current one uses "zi" for "ji", which still may be confusing for many, especially for English-based students. 


> but maybe it can be improved by adding additional notes like "X, reads as Y"

I think that thanks to introducing "di" and "du" You've solved the confusion well. I agree that it would be great to have clearly visible information how those confusing "di/du" things are pronounced, it could definitely be of a great help - given the information will be clearly visible on the screen.

When it comes to "zi", however, I think it could be great to restore the previous "ji" because it eliminates all the confusion without creating new ones. Thanks to "di" one can now be aware that it refers to ぢ, so there is no need to "zi", at least imho.

**Comment by madpipeline** [comment]
I use several apps to learn Japaneze: Duolingo, Kanji Study, HeyJapan, renshu, Easy Japanese, Human Japanese Lite, Kakugo, Nani?, Kaku and others I've tried along the way.

In all of them I've seen the romanji for じ as `ji`, and for ぢ as either `ji` or `dji` (some have `di` as well).

When I saw `zi` in this app I was completely lost as to what this kana should be.

Checking Wikipedia on the matter it uses `ji` for both and uses same marks for phonetics: 

https://en.wikipedia.org/wiki/Hiragana
https://en.wikipedia.org/wiki/Shi_(kana)
https://en.wikipedia.org/wiki/Chi_(kana)

**Comment by ThomasLeigh** [comment]
Oh, I like "dji", clever :) . Seems to be less confusing than sole "di".
